src/Microsoft.Graph.Core/Requests/Content/BatchResponseContent.cs

```diff
@@ -25,12 +25,18 @@ public class BatchResponseContent
         /// </summary>
         public ISerializer Serializer { get; private set; }
 
+        /// <summary>
+        /// Gets a <see cref="IResponseHandler"/> for handling responses.
+        /// </summary>
+        public IResponseHandler ResponseHandler { get; private set; }
+
         /// <summary>
         /// Constructs a new <see cref="BatchResponseContent"/>
         /// </summary>
         /// <param name="httpResponseMessage">A <see cref="HttpResponseMessage"/> of a batch request execution.</param>
         /// <param name="serializer">A serializer for serializing and deserializing JSON objects.</param>
-        public BatchResponseContent(HttpResponseMessage httpResponseMessage, ISerializer serializer = null)
+        /// <param name="responseHandler">A <see cref="IResponseHandler"/> for handling responses..</param>
+        public BatchResponseContent(HttpResponseMessage httpResponseMessage, ISerializer serializer = null, IResponseHandler responseHandler = null)
```

Comment:
This was my mistake. This should be resolved in the latest update.
I believe it would be very useful to provide the option of using a custom response handler. In the event there are payloads that we might not have catered for the SDK user would be able to find a work around by plugging it in.